packages/modul-components/src/components/responsive-table/responsive-table.ts

```diff
@@ -177,7 +177,26 @@ export class MResponsiveTable extends ModulVue {
 
     @Watch('headRows', { immediate: true, deep: true })
     public onHeadRowsChange(headRows: MTableHeadRows): void {
-        this.headRowsFilterAndSort = headRows;
+        if (headRows && !(this.columns && this.columns.length)) {
```

Comment:
💡  columns est required, pas besoin du  this.columns dans la condition

Reply:
et j'ajouterait  this.columns.length !== 0 c'est plus clair selon moi que this.columns.length

Reply:
La prop columns n'est pas required